dispatcher.go

```diff
@@ -57,20 +57,22 @@ type Dispatcher interface {
 
 // Config specifies the parameters of a new RPC constructed via New.
 type Config struct {
-	Name      string
+	Name string
+
 	Inbounds  []transport.Inbound
-	Outbounds transport.Outbounds
+	Outbounds Outbounds
 
 	// Filter and Interceptor that will be applied to all outgoing and incoming
 	// requests respectively.
 	Filter      transport.Filter
 	Interceptor transport.Interceptor
 
-	// TODO FallbackHandler for catch-all endpoints
-
 	Tracer opentracing.Tracer
 }
 
+// Outbounds encapsulates a service and its outbounds
+type Outbounds map[string]transport.Outbounds
```

Comment:
For a separate diff: While we're at it, we may want to make `type Inbounds
[]transport.Inbound` in the `yarpc` package too. 

Created https://github.com/yarpc/yarpc-go/issues/414 to address this later.